jans-cedarling/bindings/cedarling_python/tests/config.py

```diff
@@ -1,46 +1,50 @@
-    '''
+def load_bootstrap_config():
+    """
+    Loads the bootstrap configuration with predefined settings.
+    The policy store location can be optionally set.
+    """
 
-    # Create policy source configuration
-    # NOTE yaml is only used for test fixtures. Real imports use json.
-    policy_store_location = join(TEST_FILES_PATH, "policy-store_ok.yaml")
+    def _load_config(policy_store_location=None, log_type="std_out", log_ttl=None):
+        if policy_store_location is None:
+            policy_store_location = os.path.join(TEST_FILES_PATH, "policy-store_ok.yaml")
 
-    bootstrap_config = BootstrapConfig({
-        "application_name":"TestApp",
-        "policy_store_id":"asdasd123123",
-        "policy_store_local_fn":policy_store_location,
-        "jwt_sig_validation":"disabled",
-        "jwt_status_validation":"disabled",
-        "at_iss_validation":"disabled",
-        "at_jti_validation":"disabled",
-        "at_nbf_validation":"disabled",
-        "idt_iss_validation":"disabled",
-        "idt_sub_validation":"disabled",
-        "idt_exp_validation":"disabled",
-        "idt_iat_validation":"disabled",
-        "idt_aud_validation":"disabled",
-        "id_token_trust_mode":"none",
-        "userinfo_iss_validation":"disabled",
-        "userinfo_aud_validation":"disabled",
-        "userinfo_sub_validation":"disabled",
-        "userinfo_exp_validation":"disabled",
-        "log_type":"std_out"
-    })
+        return BootstrapConfig({
+            "CEDARLING_APPLICATION_NAME": "TestApp",
+            "CEDARLING_POLICY_STORE_ID": "a1bf93115de86de760ee0bea1d529b521489e5a11747",
+            "CEDARLING_POLICY_STORE_LOCAL_FN": policy_store_location,
+            "CEDARLING_USER_AUTHZ": "enabled",
+            "CEDARLING_WORKLOAD_AUTHZ": "enabled",
+            "CEDARLING_WORKLOAD_BOOLEAN_OPERATION": "AND",
+            "CEDARLING_JWT_SIG_VALIDATION": "disabled",
+            "CEDARLING_JWT_STATUS_VALIDATION": "disabled",
+            "CEDARLING_AT_ISS_VALIDATION": "disabled",
+            "CEDARLING_AT_JTI_VALIDATION": "disabled",
+            "CEDARLING_AT_NBF_VALIDATION": "disabled",
+            "CEDARLING_AT_EXP_VALIDATION": "disabled",
+            "CEDARLING_IDT_ISS_VALIDATION": "disabled",
+            "CEDARLING_IDT_SUB_VALIDATION": "disabled",
+            "CEDARLING_IDT_EXP_VALIDATION": "disabled",
+            "CEDARLING_IDT_IAT_VALIDATION": "disabled",
+            "CEDARLING_IDT_AUD_VALIDATION": "disabled",
+            "CEDARLING_USERINFO_ISS_VALIDATION": "disabled",
+            "CEDARLING_USERINFO_AUD_VALIDATION": "disabled",
+            "CEDARLING_USERINFO_SUB_VALIDATION": "disabled",
+            "CEDARLING_USERINFO_EXP_VALIDATION": "disabled",
+            "CEDARLING_ID_TOKEN_TRUST_MODE": "none",
+            "CEDARLING_LOG_TYPE": log_type,
+            "CEDARLING_LOG_TTL": log_ttl,
+        })
 
-    return bootstrap_config
+    return _load_config
```

Comment:
Why we return `_load_config` here?

Reply:
Probably we no need this fixture if it works in this way...

We can just define the function in this file and import it

Reply:
fixed here: [1dcc381](https://github.com/JanssenProject/jans/pull/10350/commits/1dcc381ed868dc8e5d98b472cf2d657cd06b7fa5)